src/main/java/br/com/studiesMaterials/handlers/ArticleHandler.java

```diff
@@ -0,0 +1,101 @@
+package br.com.studiesMaterials.handlers;
+
+import br.com.studiesMaterials.dao.ArticleDao;
+import br.com.studiesMaterials.db.DataBase;
+import br.com.studiesMaterials.web.api.schemas.ArticleSchema;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.google.gson.Gson;
+import java.sql.*;
+
+public class ArticleHandler implements ArticleDao {
+    @Override
+    public APIGatewayProxyResponseEvent create(APIGatewayProxyRequestEvent input) {
+        APIGatewayProxyResponseEvent responseEvent = new APIGatewayProxyResponseEvent();
+        try(Connection conn = DataBase.connection()) {
+            assert conn != null;
+            Statement statement = conn.createStatement();
+
+            String studentId =  input.getPathParameters().get("student_id");
+
+            ArticleSchema data = convertBody(input.getBody());
+
+            String sql = String.format(
+                    "INSERT INTO public.article (student_id, subject, link) " +
+                            "VALUES ('%s', '%s', '%s')", studentId, data.subject, data.link
+            );
+
+            statement.executeUpdate(sql);
+
```

Comment:
Seria legal fechar as conexões do banco depois do executeUpdate
```suggestion
conn.close()
```